operations/mimir/multi-zone-distributor.libsonnet

```diff
@@ -0,0 +1,172 @@
+// This file contains the experimental configuration to deploy distributors in multi-AZ.
```

Comment:
Please import this file in `operations/mimir/mimir.libsonnet`, right after the line `(import 'multi-zone.libsonnet')`. It's OK keeping this stuff in a separate file, because `multi-zone.libsonnet` is already quite big.